docker/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM debian:bullseye-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+RUN echo '\
+Acquire::http::No-Cache true;\
+APT::Get::Assume-Yes "true";\
+APT::Install-Recommends "false";\
+APT::Install-Suggests "false";\
+' > /etc/apt/apt.conf.d/99custom
+
+RUN apt-get update && apt-get upgrade
+RUN apt-get install curl
+RUN apt-get install ca-certificates
```

Comment:
why do we need to pull unnecessary dependencies?

Reply:
This is needed to download the release from GitHub. We could remove then if you chose.

Reply:
But we can just copy the binary into container, no? :)

Reply:
Indeed we can :)